blockchain/utxocache.go

```diff
@@ -232,7 +232,7 @@ type utxoBatcher interface {
 	// getEntries attempts to fetch a serise of entries from the UTXO view.
 	// If a single entry is not found within the view, then an error is
 	// returned.
-	getEntries(outpoints map[wire.OutPoint]struct{}) (map[wire.OutPoint]*UtxoEntry, error)
```

Comment:
Ah I guess "conventional " wisdom is that `struct{}` or `interface{}` doesn't add over head, but that doesn't factor in the map bucket overhead 🤔

Reply:
Yeah I've noticed the map bucket overhead messing with the size calculation.

No need for a map here anyways so I think a slice is a strict improvement.